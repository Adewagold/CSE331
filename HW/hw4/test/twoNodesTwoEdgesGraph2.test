# Tests adding two nodes, two edges between two nodes to graph.

# Create a graph
CreateGraph two_nodes_two_edges_graph2

# Add two nodes
AddNode two_nodes_two_edges_graph2 a
AddNode two_nodes_two_edges_graph2 b

# Add two edges
AddEdge two_nodes_two_edges_graph2 a b AB
AddEdge two_nodes_two_edges_graph2 b a BA

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_two_edges_graph2
ListChildren two_nodes_two_edges_graph2 a
ListChildren two_nodes_two_edges_graph2 b