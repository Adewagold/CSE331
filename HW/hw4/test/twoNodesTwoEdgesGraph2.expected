# Tests adding two nodes, two edges between two nodes to graph.

# Create a graph
created graph two_nodes_two_edges_graph2

# Add two nodes
added node a to two_nodes_two_edges_graph2
added node b to two_nodes_two_edges_graph2

# Add two edges
added edge AB from a to b in two_nodes_two_edges_graph2
added edge BA from b to a in two_nodes_two_edges_graph2

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_two_edges_graph2 contains: a b
the children of a in two_nodes_two_edges_graph2 are: b(AB)
the children of b in two_nodes_two_edges_graph2 are: a(BA)